gradle/build.versions.toml

```diff
@@ -39,6 +40,7 @@ mockito-core = "org.mockito:mockito-core:1.9.5"
 nebula-info = "com.netflix.nebula:gradle-info-plugin:11.3.3"
 reflections = "org.reflections:reflections:0.9.12"
 shadow-plugin = "com.github.breskeby:shadow:3b035f2"
+snakeyaml = { group = "org.yaml", name = "snakeyaml", version = { strictly = "2.0" } }
```

Comment:
What is the behavior of "strictly"? Is this equivalent to `force` in a normal dependency declaration?

Reply:
Basically this means `Any version not matched by this version notation will be excluded.` We use strictly here as we use this dependency in a constraint we declare in built-tools-internal to handle transitive dependencies there. `force = true` doesn't allow us doing that